lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralLinearModelAlgorithm.cxx

```diff
@@ -368,6 +371,9 @@ void GeneralLinearModelAlgorithm::initializeDefaultOptimizationAlgorithm()
   TNC* tnc = dynamic_cast<TNC *>(solver_.getImplementation().get());
   if (tnc)
     tnc->setIgnoreFailure(true);
+  Cobyla* cobyla = dynamic_cast<Cobyla *>(solver_.getImplementation().get());
+  if (cobyla)
+    cobyla->setIgnoreFailure(true);
```

Comment:
```suggestion
  OptimizationAlgorithm singleStartSolver = OptimizationAlgorithm::Build(solverName);
  TNC* tnc = dynamic_cast<TNC *>(singleStartSolver.getImplementation().get());
  if (tnc)
    tnc->setIgnoreFailure(true);
  Cobyla* cobyla = dynamic_cast<Cobyla *>(singleStartSolver.getImplementation().get());
  if (cobyla)
    cobyla->setIgnoreFailure(true);
  solver_ = MultiStart(singleStartSolver, 10);
```

Here is an example showing how to use the new `MultiStart` constructor within GLM.
In fact, you would also delete the line 
``solver_ = OptimizationAlgorithm::Build(solverName);``
but GitHub doesn't allow suggestions for that line because it is too far removed from the changes of this PR.

Reply:
I don't think you need to make any changes to `run`, `setOptimizationBounds` or `setOptimizationAlgorithm` if you only want to change the default behavior.

Reply:
What you propose is quite different from what I had in mind: to let the user choose the base solver and to implement the multi-start strategy based on it if the relevant ResourceMap key is set to True. I think I will let you change the code once this PR is merged, as it contains many other things I would like to see in master ASAP.

Do you agree with it?

BTW I am not fond of hard-coded constants such as the 10 as a second argument of MultiStart. You should definitely use a dedicated key in ResourceMap or something else to let the user choose by himself the value.

Reply:
See my previous comment on the use of MultiStart in this PR and my comments about your PR. I will leave the code as is, waiting for a better evolution of MultiStart to **consider** its use in GLMAlgo in a different way than what is proposed here.